src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/Jira.kt

```diff
@@ -16,9 +18,15 @@ class Jira(
     val address: URI,
     val jmxClients: List<JmxClient> = emptyList()
 ) : MeasurementSource {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
     override fun gatherResults() {
+        if (nodes.isEmpty()) {
```

Comment:
Yikes, good catch. `Executors.newFixedThreadPool` `@throws IllegalArgumentException if {@code nThreads <= 0}`
That's my bad for skipping tests in https://github.com/atlassian/aws-infrastructure/pull/59